README.md

```diff
@@ -23,26 +23,6 @@ The goal is to enable NRI support in the most commonly used OCI runtimes,
 [containerd](https://github.com/containerd/containerd) and
 [CRI-O](https://github.com/cri-o/cri-o).
 
-## Background
```

Comment:
Should we still keep a revisited short 'Background' or 'Introduction' section which briefly tells in a few sentences what NRI is ?

Reply:
Well, we have the Goal section. Maybe that's good enough then.